app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainActivity.java

```diff
@@ -0,0 +1,87 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.example.lapnguyen.weather_3.R;
+import com.example.lapnguyen.weather_3.api.ServiceGenerator;
+import com.example.lapnguyen.weather_3.api.WeatherService;
+import com.example.lapnguyen.weather_3.data.model.Weather;
+import com.example.lapnguyen.weather_3.data.source.MainRepository;
+import com.example.lapnguyen.weather_3.data.source.MainRepositoryImplement;
+import com.example.lapnguyen.weather_3.screen.settingscreen.SettingsActivity;
+import com.example.lapnguyen.weather_3.utils.Navigator;
+import org.w3c.dom.Text;
```

Comment:
ERROR: Unused import: org.w3c.dom.Text. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports